Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/ConnectionPage.tsx

```diff
@@ -65,6 +65,8 @@ export default function ConnectionPage(props: IProps) {
 
     const [asc, setAsc] = React.useState<boolean>(true);
     const [sortKey, setSortKey] = React.useState<string>('AssetName');
+    // state for tooltip on delete existing button
+    const [hoverOnDelete, setHoverOnDelete] = React.useState<boolean>(false);
```

Comment:
The issue with this is if you have 2 disabled buttons hovering over either will make both tooltips occur.
Solution would be to create a new React component with a `hover` state. Move everything in `Content` into there and add anything that needs to connect out (like `ID` and `onClick`) into there.

Also you don't need the `div` that may cause issues with consistent styling as well.